### PR TITLE
Publish updates from master

### PR DESCRIPTION
**Pull request by usha-mandya**
Publish the latest changes from master

### CODE REVIEW

File: _data/glossary.yaml [hunk 1]
```diff
@@ -134,9 +132,9 @@ filesystem: |
 
   Examples :
 
-  - Linux : ext4, aufs, btrfs, zfs
+  - Linux : overlay2, extfs, btrfs, zfs
```
Comment (thaJeztah) [inline _data/glossary.yaml:135]: `overlay2` is probably wrong in this context, as that's the name of a storage driver in Docker Engine, but the name in the kernel would be `overlayfs` 🤔

**Review by thaJeztah — APPROVED** [review]
LGTM (left one comment for a follow-up)

File: _data/glossary.yaml (hunk 1 above)
Comment (StefanScherer) [reply _data/glossary.yaml:135]: That‘s my fault 😅

Comment (thaJeztah) [reply _data/glossary.yaml:135]: no worries! we need to consider what we want to do with the glossary altogether; I guess it's not viewed too often

**Pull request merged**